src/loggger.ts

```diff
@@ -2,18 +2,18 @@ export class Logger {
 	protected buffer: string = "";
 	protected tag: string = '';
 	protected time: boolean = false;
-	
```

Comment:
Do not commit files with only code style changes.

Reply:
I've done style changes in my very first commit of this PR, If I delete that commit know I think I will have some conflicts